### DISCUSSION TITLE
sqlmodel.update Pylance error

### DISCUSSION THREAD

**Discussion by yahahawork** [fastapi/sqlmodel#758] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from sqlmodel import and_, col, select, update

ids = [int(i) for i in host_ids.split(",")]
filters = and_(
    col(HostModel.uid).in_(ids),
    col(HostModel.busy) == HostStatus.IDLE.value
)
stmt = (
    update(HostModel)
    .filter(filters)
    .values(
        busy=HostStatus.BUSY.value,
        task_id=task.uid
    )
)
session.exec(stmt)
```

### Description

Pylance shows not match any overload of exec

### Operating System

Windows

### SQLModel Version

0.0.14

### Python Version

3.12.1

**Comment by YuriiMotov** [comment]
There is an open issue for this: https://github.com/fastapi/sqlmodel/issues/909
And PR: https://github.com/fastapi/sqlmodel/pull/1342
Let's track it there

I'll close this discussion.
